### PR TITLE
security: CVE-2020-7598

### PR DESCRIPTION
**Pull request by artamonovkirill**
A dependabot alert raised by GitHub on my fork of this repo: https://github.com/artamonovkirill/problem-specifications/network/alert/yarn.lock/minimist/open
Also, the same vulnerability is reported by `yarn audit`:
```
yarn audit v1.22.10
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.2.1 <1.0.0 || >=1.2.3                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ ajv-cli                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ ajv-cli > ajv-pack > js-beautify > mkdirp > minimist         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1179                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ minimist                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.2.1 <1.0.0 || >=1.2.3                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ ajv-cli                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ ajv-cli > minimist                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1179                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
2 vulnerabilities found - Packages audited: 37
Severity: 2 Low
✨  Done in 0.67s.
```

While it's severity is low and the attack surface is probably non-existent, my preference is - if it's a matter of a simple version bump - to fix such security warnings to have an uncluttered view when more severe vulnerabilities are reported.

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
As there is only one dependency in the package.json file (`ajv-cli`), maybe it would be better to update that package to the latest version (3.3.0 instead of 2.1.0)?

**Comment by artamonovkirill** [comment]
@ErikSchierboom, I can update it.
Would it be a good idea to configure a tool like [renovate](https://renovate.whitesourcesoftware.com/) or [dependabot](https://dependabot.com/) for this repo?

**Comment by SleeplessByte** [comment]
The attack surface on this is non existent.

While I really appreciate the work, it's favourable to know _why_ a CVE affects our repos and I'd request in the future that PR descriptions for PRs like these are not empty but hold information on the importance of merging.

We can turn on dependablebot, but in my experience it will be a lot of noise. We should turn on security vulnerability notifications tho.

**Comment by artamonovkirill** [comment]
@SleeplessByte , 

>  I'd request in the future that PR descriptions for PRs like these are not empty but hold information on the importance of merging

Sorry for that, I've added a description.

File: package.json [hunk 1]
```diff
@@ -7,10 +7,13 @@
...
   "dependencies": {
-    "ajv-cli": "^2.1.0"
+    "ajv-cli": "^3.3.0"
   },
   "scripts": {
     "test": "ajv -s canonical-schema.json -d \"exercises/*/canonical-data.json\"",
     "test-one": "ajv -s canonical-schema.json -d"
+  },
+  "resolutions": {
+    "minimist": "^1.2.3"
```
[the remark this answers is not in the document]
Comment (artamonovkirill) [reply package.json:17]: Unfortunately, yes - running `yarn audit` on commit 23683bd still yields the same 2 vulnerabilities.

**Review by ErikSchierboom — APPROVED** [review]

**Review by SleeplessByte — APPROVED** [review]
@exercism/reviewers just need one more

**Review by neenjaw — APPROVED** [review]
Seems reasonable, nice work

**Comment by neenjaw** [comment]
@SleeplessByte I'll leave to you or @ErikSchierboom to merge. 🎉

**Comment by SleeplessByte** [comment]
Don't need to tell me twice 😁😁

**Pull request merged**